### ISSUE TITLE
vague usage example

### ISSUE THREAD

**Issue by mouhsinelonly** [jpodwys/superagent-cache#2]
the usage example is vague should i use this extensions alongside with superagent or standalone .
i'm using browserify import how can us superagent-cache with it ?

example with suerpagent :
`
import request from superagent;
request.get....`

**Comment by jpodwys** [comment]
Sounds like you have two questions, but correct me if I'm wrong.

1- Should I use this extension alongside superagent or as a standalone?

You can have superagent-cache generate a superagent instance for you or patch your existing one. The [Various ways of requiring superagent-cache](https://github.com/jpodwys/superagent-cache#various-ways-of-requiring-superagent-cache) section from the documentation outlines these use cases.

2- How can I use browserify import with superagent-cache?

```javascript
import superagentCache from './path/to/superagent-cache';
var superagent = superagentCache();

superagent.get...
```

**Comment by jpodwys** [comment]
Does this answer your question?

**Comment by mouhsinelonly** [comment]
i'm using npm can also use

import superafentCache from 'superagent-cache';

?

**Comment by jpodwys** [comment]
Since you're using `import {module} from {location}` syntax, I assume you're using ECMAScript 6. As a result of using this syntax instead of the `require('module-name')` syntax, you must specify your relative file path in the `{location}` param rather than simply providing the module name you want to use.

See the accepted answer to [this Stack Overflow question](http://stackoverflow.com/questions/28387775/ecmascript-6-module-requiring-how-does-it-work) for a more detailed explanation.

Therefore, the code I gave you in my first comment should work just fun as long as your relative path to superagent-cache is correct. If you're using NPM, it will be located in your project's `node_modules` folder.

**Comment by mouhsinelonly** [comment]
yes i'm using NPM , require('module-name') works but nothing is cached and data is retrieved each time i make an HTTP Request ?

**Comment by jpodwys** [comment]
OK I'm able to reproduce this. Working on a fix and will let you know when it's ready.

**Comment by jpodwys** [comment]
Fixed and published to NPM in `1.0.2`.

**Comment by jpodwys** [comment]
Is this working for you?
